guidelines/mwm-sadd.md

```diff
@@ -150,32 +152,43 @@ May include a partial schema - the included attributes will be updated in the Wo
 URL: `/tasks`
 Endpoints:
 - Retrieve task by ID.
-- Provide task result by ID: used by app servers to perform callbacks.
 - Retrieve tasks (filter by current/time period)
 
 _More details to be added._
 
 ### Internal Modules
 
-#### Payload Listener
-The Payload Listener monitors an input queue. The MIG (or a custom ingestion service) will add an event to that queue when new data is sent to the system.
+#### Event Broker Adapter
+And the adapter communicates with an event broker.
+
+The adapter will listen on multiple queues, as  follows:
+
+| Queue Name | Publisher | Consumer |
+|------|------|------|
+|md.workflow.request|Informatics Gateway|Workflow Executer
+|md.workflow.task_dispatch|Workflow Executer|Task Executer
+|md.workflow.task_callback|App Servers|Task Executer
+|md.export.complete|Workflow Executer|Informatics Gateway
```

Comment:
Should `md.export.complete` be coming from IG as follows?

| Queue Name | Publisher | Consumer |
|------|------|------|
| md.export.request | Workflow Executer | Informatics Gateway |
|md.export.complete|Informatics Gateway |Workflow Executer|

Reply:
Hi, I think I saw it somewhere which is why I included it – the workflow manager doesn't need it, but it could be a useful event to have in the future.

Reply:
For `md.export.complete`, I expect this to come from IG (or any export services) signaling the WM of a complete export, correct? This is what IG emits currently.  IG also listens to `md.export.request` for export tasks.